.editorconfig

```diff
@@ -44,6 +70,9 @@ end_of_line = lf
 charset = utf-8-bom
 
 [*.cs]
-dotnet_diagnostic.CA1304.severity = error
-dotnet_diagnostic.CA1305.severity = error
-dotnet_diagnostic.CA1310.severity = error
+# Temporarily disabling, waiting on PR feedback to re-enable
+dotnet_diagnostic.CA1310.severity = none
```

Comment:
💡 Setting diagnostic severity is more efficient if it's done in a globalconfig instead of editorconfig. See https://github.com/dotnet/aspnetcore-tooling/pull/3087 for an example of how I converted aspnetcore-tooling from rulesets to globalconfig.